packages/components/src/box-control/test/index.js

```diff
@@ -164,6 +192,9 @@ describe( 'BoxControl', () => {
 
 	describe( 'Unlinked sides', () => {
 		it( 'should update a single side value when unlinked', async () => {
+			const user = userEvent.setup( {
+				advanceTimers: jest.advanceTimersByTime,
+			} );
 			let state = {};
 			const setState = ( newState ) => ( state = newState );
```

Comment:
Should we use the wrapper testing component in the remaining places where we "mock" the state?

```jsx
const Example = () => {
  const [ state, setState ] = useState();
  
  return (
	  <BoxControl
		  values={ state }
		  onChange={ ( next ) => setState( next ) }
	  />
  );
};
```

Reply:
Sure thing! This is the only bit we haven't touched in this refactor PR 😅 

I've addressed your suggestion in https://github.com/WordPress/gutenberg/pull/45968/commits/c3f0a3718de23a07ade06d20b878701795d6046d where I removed the wrapper component mock redundancy and improved related assertions to be more user-centric.

Let me know what you think!